prime-router/src/main/kotlin/transport/FTPSTransport.kt

```diff
@@ -167,13 +166,13 @@ class FTPSTransport : ITransport, Logging {
             } catch (e: IOException) {
                 if (ftpsClient.isConnected) {
                     try {
-                        println("Closing connection due to exception")
+                        context.logger.warning("Closing connection due to exception")
```

Comment:
Use log4j logger.warn instead.

Reply:
@carlosfelix2 just double-checking this -- this method uses `context.logger` a few lines up when logging an issue; it seems reasonable to do the same here, if only to keep these messages together

(same goes for comment a few lines down)